src/components/common/InvestorsLoginSidebar.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import step1img from '../../images/steps (6).svg'
+
+class LoginSideBar extends Component {
+  render() {
+    return (
+            <div className='login-sidbar'>
+              <div className="sidebar-heading">
+                <h1>{this.props.heading} <span className='green-span'></span></h1>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['heading' is missing in props validation](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5554130)